src/index.html

```diff
@@ -11,7 +11,149 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="input">
+          <label for="surname">Surname:</label>
+          <input
+          type="text"
+          name="Surname"
+          id="surname">
```

Comment:
```suggestion
          <input
            type="text"
            name="Surname"
            id="surname"
          >
```
Let's fix the indentations. You may read about it here
https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping